CHANGELOG.md

```diff
@@ -6,14 +6,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
 ### Added
 
 - Add tests.
+- Add w-100 for slider.
```

Comment:
I don't know if this is only because of the slider.

Reply:
What do you suggest?